test/js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,117 @@
+describe('ez-selection-activate-element', function() {
+    let element;
+    let elementSubsetElementChecked;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+        elementSubsetElementChecked =  fixture('TestFixtureSubsetElementChecked');
+    });
+
+    function checkAndclickOneElement(elem, selector, isChecked) {
```

Comment:
it should `checkAndClickOneElement` but I find weird to have a method called `checkSomething` that takes a `isCheck` parameter (so it does not check anything when it's false)